src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -92,6 +92,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.10.*" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.Core" Version="2.8.0" />
```

Comment:
At the time of writing, this package version is currently only on myget.org. We'll need to publish it to nuget.org prior to releasing this extension version.